cssbeautify/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for CSS Beautify v0.3.1
+// Project: https://github.com/senchalabs/cssbeautify
+// Definitions by: rictic <https://github.com/rictic>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface Options {
+  /**
+   * A string used for the indentation of the declaration (default is 4
+   * spaces).
+   */
+  indent?: string;
+  /**
+   * Defines the placement of open curly brace, either end-of-line (default)
+   * or separate-line
+   */
+  openbrace?: 'end-of-line'|'separate-line';
+
+  /**
+   * Always inserts a semicolon after the last ruleset(default is false)
+   */
+  autosemicolon?: boolean;
+}
+declare namespace beautify {}
```

Comment:
`css-beautify` uses [`module.exports = ...`](https://github.com/senchalabs/cssbeautify/blob/master/cssbeautify.js#L463). It's not an ES6 module. Declaring a namespace might fool the compiler into thinking it can treat it as an ES6 module, but this is a bad idea.
More info [here](https://stackoverflow.com/questions/39415661/why-cant-i-import-a-class-or-function-with-import-as-x-from-y).